.github/workflows/sdk-build.yml

```diff
@@ -32,6 +32,8 @@ jobs:
         run: pydocstyle --match-dir='^(?!build$).*' --match='^(?!(__init__\.py|setup\.py$)).*\.py$' src
       - name: Lint with pydoclint
         run: pydoclint --exclude='.*/build/.*' src
+      - name: Lint with pylint
+        run: pylint src --disable=all --enable=C0103 --ignore=build
       - name: Lint with flake8
```

Comment:
If we're adopting `pylint` then should we remove all references to `flakes8` rather than having a mix of both?

* Remove step that runs `flake8` in this workflow
* Remove `pyflakes` dependency from requirements.txt
* Update `.vscode/settings.json` to include setting that enables Pylint in VS Code 
* Replace `flakes8` with `pylint` in "Code Standards" section of CONTRIBUTING.md

Reply:
I definitely think this is worth considering, but in the interest of avoiding scope creep, maybe this can be created as a separate issue?

Reply:
https://github.com/zowe/zowe-client-python-sdk/issues/319

I'll leave it up to @pem70 to handle that issue as part of this PR or as part of a new one 😋

Reply:
I will handle this as part of this PR